src/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './App.css';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <>
+      <Display>0</Display>
+      <ButtonPanel>0</ButtonPanel>
+    </>
+  );
+}
+
+export default App;
```

Comment:
[OPTIONAL]

- [ ] Please let's make use of ES6 syntax.
- [ ] Kindly delete all unnecessary files from your repo. e.g `reportWebVitals.js`, `setupTests.js`, `App.test.js`. All the image files in the `public` directory also fall in this category. This is a mandatory requirement from milestone-1, I'm making it optional in this milestone because it must have fallen through the cracks in the initial milestone-1 review.  I strongly encourage you to implement this change.